Basic-Video-Chat-With-ForegroundServices-Java/build.gradle

```diff
@@ -0,0 +1,9 @@
+// Top-level build file where you can add configuration options common to all sub-projects/modules.
+plugins {
+    id 'com.android.application' version '7.1.2' apply false
+    id 'com.android.library' version '7.1.2' apply false
+}
```

Comment:
These versions are really old, aren't they?

Reply:
The sample was built on top of a copy of Basic-Video-Chat. Yes, these versions are quite old. I can maybe bump up it for all samples

Reply:
Upgraded to 8.7